rep-2005.rst

```diff
@@ -578,6 +579,29 @@ All items on the list for the next distro should already be released into the ro
     * `rqt_tf_tree/rqt_tf_tree <https://index.ros.org/p/rqt_tf_tree/>`_
     * `rqt_topic/rqt_topic <https://index.ros.org/p/rqt_topic/>`_
 
+  * micro-ROS
+
+    * `micro-ROS/benchmarking <https://github.com/micro-ROS/benchmarking>`_
+    * `micro-ROS/docker <https://github.com/micro-ROS/docker>`_
+    * `micro-ROS/freertos_apps <https://github.com/micro-ROS/freertos_apps>`_
+    * `micro-ROS/micro_ros_arduino <https://github.com/micro-ROS/micro_ros_arduino>`_
+    * `micro-ROS/micro_ros_espidf_component <https://github.com/micro-ROS/micro_ros_espidf_component>`_
+    * `micro-ROS/micro_ros_setup <https://github.com/micro-ROS/micro_ros_setup>`_
+    * `micro-ROS/micro_ros_zephyr_module <https://github.com/micro-ROS/micro_ros_zephyr_module>`_
+    * `micro-ROS/micro-ROS-Agent <https://github.com/micro-ROS/micro-ROS-Agent>`_
+    * `micro-ROS/micro-ROS-bridge_RPI <https://github.com/micro-ROS/micro-ROS-bridge_RPI>`_
```

Comment:
I think that this is not supported any more

Reply:
Removed micro-ROS-bridge_RPI.